src/PlaywrightRunner.ts

```diff
@@ -95,15 +96,24 @@ const getDevices = (
   return resultDevices
 }
 
+const setJestTimeout = (configTimeout?: number) => {
+  if (configTimeout) {
+    return configTimeout
+  }
+  return process.env.PWDEBUG === '1'
```

Comment:
```suggestion
  return process.env.PWDEBUG
```

Checking for a truthy value is fine here.